fpdc_client/base.py

```diff
@@ -36,21 +36,37 @@ class FPDC:
 
     Args:
         url (str, optional): The URL of the FPDC server.
+        staging (boolean): Set to True to use FPDC staging instance
     """
 
     app_id = "fpdc-client"
 
-    def __init__(self, url=DEFAULT_URL):
+    def __init__(self, url=DEFAULT_URL, staging=False):
```

Comment:
Actually, you can detect if you're in staging or prod by looking at the url. If it's `fpdc.stg.fp.o` you're in staging, if it's `fpdc.fp.o` you're in prod. Then you can preset the issuer accordingly. If the URL is neither, you don't set the issuer (or set it to None) and `login()` must use the `auth_file` arg.
What do you think?